src/cmd-buildextend-azure

```diff
@@ -182,7 +182,10 @@ def run_ore(args, build):
     checksum = sha256sum_file(azure_vhd_path)
     size = os.path.getsize(azure_vhd_path)
 
-    build.meta['azure'] = {'image': build.azure_vhd_name}
+    build.meta['azure'] = {
+        'image': build.azure_vhd_name,
+        'path': f"{args.container}/imagebucket/{build.azure_vhd_name}",
```

Comment:
I think what you want is something like `${args.storage_account}.blob.core.windows.net/${args.container}/{build.azure_vhd_name}` ?

Reply:
i'm not familiar with variables. So can we include the example of the end results.

Reply:
Does "path" here have a meaning?  Why not e.g. have this be:

```
{"storage_account": "${storage_account},
  "container": "${container}"
  ...
}
```

Sorry for my unfamiliarity with Azure/GCP storage terminology, but imagine we were doing this same thing for AWS (i.e. we stopped uploading AMIs and just had something in an s3 bucket).  There are a few ways we could represent the information; probably simplest to say `s3://bucket/object` but in theory we could also say e.g.:

```
{"bucket": "${bucketname}",
  "objectname": "${objectname}" }
```

etc.

Reply:
It would be easiest if looked like `https://rhcospipelineimage
s2.blob.core.windows.net/imagebucket/rhcos-420devel.8.20190719.0.vhd`

Reply:
Changed to URL. 
Now we get something like:
```
<SNIP>
  "azure": {
    "image": "rhcos-420bh.8.20190723.1.vhd",
    "url": "https://rhcoshorcruxes.blob.core.windows.net/imagebucket/rhcos-420bh.8.20190723.1.vhd"
  },
  "gcp": {
    "image": "rhcos-420bh-8-20190723-1",
    "url": "https://storage.googleapis.com/rhcos-devel/devel/rhcos/420bh.8.20190723.1.tar.gz"
  }
}
```